sematic/ui/package.json

```diff
@@ -54,7 +54,11 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "lint": "eslint --config node_modules/eslint-config-react-app/index.js --ext .ts,.tsx --max-warnings=0 ./src"
+    "lint:app": "eslint --config node_modules/eslint-config-react-app/index.js --ext .ts,.tsx --max-warnings=0 ./src",
```

Comment:
Please add the ui linting to the `pre-commit` Makefile target in a follow-up PR. We want to avoid including lint errors in the UI, just like we do with backend code.